pom.xml

```diff
@@ -377,7 +532,43 @@
         <profile>
             <id>basic</id>
             <modules>
-                <module>tests/basic</module>
+                <module>tests-configuration-batch</module>
```

Comment:
Now it seems that same modules are started with and without basic profile because all basic modules are added to default modules https://github.com/hal/testsuite.next/pull/147/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R104

So maybe what about to remove basic modules from default modules [here](https://github.com/hal/testsuite.next/pull/147/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R104)?

Reply:
And keep tooling modules only there

Reply:
Ah, makes sense. Thank you for spotting this!